README.md

```diff
@@ -1,10 +1,55 @@
 MakersBnB
 =========
 
-# project spec
+We are building a short-term rental website for travellers going away. Our platform allows hosts to list the properties that they would like to rent out to guests, and for guests to pick their preferred choice of rental. The platform allows hosts to manage their diary, approve specific booking requests and more.
 
-User Stories
-----------
+## Installation
+-----
+- Fork this repository
+- ```cd``` in the forked directory
+- Run the command ```bundle install``` in the terminal
+- Run ```rackup``` in the terminal to start the server
+- Visit ```localhost:9292/listings``` in your browser 
+
+
+## How to run the tests after all the gems have been installed
+-----
+1. ```cd``` into the forked directory
+2. Run ```rspec``` in the terminal
+
+
+## Overview Functionality
+-----
+
+### The user is able to:
+  - sign-up and create an account
+  - sign-in/sign-out
+  - view all the available properties 
+  - view details about each property
+  - list a property/properties
+  - book a property
+  - see the availability
+  - choose a start-date/end-date  
+  - see the total price of the requested dates
+  - request a booking
+   
+##  Steps taken in order to build the application
+1. MVP which consisted of:
+    - spec
+    - user stories 
+    - low-fidelity wireframe
+    - database domain modeling
+    - barebones functional website
+      - list of properties
+      - view single property
+2. Installing the relevant gems
+3. Using the MVC design pattern
+4. Implementing feature and unit tests before writing any line of code
```

Comment:
There could be more about our team process here - how we chose to divide the user stories, tools we used to keep us as a team on track, but good to see the MVP up there!